src/shogun/machine/Pipeline.cpp

```diff
@@ -222,4 +223,37 @@ namespace shogun
 	{
 		return shogun::get<CMachine*>(m_stages.back().second);
 	}
+
+	void CPipeline::set_store_model_features(bool store_model)
+	{
+		get_machine()->set_store_model_features(store_model);
+	}
+
+	void CPipeline::store_model_features()
+	{
+		get_machine()->store_model_features();
+	}
+
+	CSGObject* CPipeline::clone()
+	{
+		auto result = CMachine::clone()->as<CPipeline>();
+		for (auto&& stage : m_stages)
```

Comment:
I think we could probably do this via the parameter framework clone, with a few additions for std::types. The class is not serializable anyways.
@lisitsyn what do you think?